readthedocs/doc_builder/director.py

```diff
@@ -491,7 +491,14 @@ def install_build_tools(self):
             # TODO: generate the correct path for the Python version
             # see https://github.com/readthedocs/readthedocs.org/pull/8447#issuecomment-911562267
             # tool_path = f'{self.config.build.os}/{tool}/2021-08-30/{full_version}.tar.gz'
-            tool_path = f"{self.data.config.build.os}-{tool}-{full_version}.tar.gz"
+
+            build_os = self.data.config.build.os
+            if "lts-latest" in self.data.config.build.os:
```

Comment:
```suggestion
            if build_os == "ubuntu-lts-latest":
```

Reply:
Good point. I'm not sure why I didn't do this originally 🙃